src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -506,6 +506,14 @@ async def test_maximum_concurrent_rpcs(self):
         await channel.close()
         await server.stop(0)
 
+    async def test_is_active(self):
+        if (self._server.context.is_active()):
+            pass
+
+    async def test_set_callback(self):
+        def callback(call):
+            return
+        self._server.context.add_callback(callback)
```

Comment:
We will need stronger test cases:

1. Ensure the callback is actually invoked;
2. Multiple callbacks;
3. Callback that raises.

Reply:
Thanks @lidizheng for the feedback. Are there similar tests elsewhere on which I can model my tests that you might be able to help me point to?

Reply:
I think the most similar one would be https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests_aio/unit/done_callback_test.py.

I recommend to use Bazel to debug:

```sh
# Test all cases
bazel test grpc/src/python/grpcio_tests/tests_aio/...
```